common/engine/AssetManager.js

```diff
@@ -234,7 +234,7 @@ function downloadBundle (nameOrUrl, options, onComplete) {
                     let sys = cc.sys;
                     if (sys.platform === sys.ALIPAY_GAME && sys.os === sys.OS_ANDROID) {
                         let resPath = unzipPath + 'res/';
-                        if (fs.accessSync({path: resPath})) {
+                        if (fs.accessSync({path: resPath}).success) {
```

Comment:
the usage is conflit with their current docs https://opendocs.alipay.com/mini/api/025027

Reply:
according the docs
https://www.yuque.com/zhifubaoxiaoyouxi-zromu/ts2ry9/4e26f2203cdb4c74f80c7fd37382ad0f

Reply:
this is the earliest docs they provided several years ago, is it still valid for now ?

Reply:
They said to use the modification suggestions they provided.

Reply:
They said we can refer to the usage method at https://www.yuque.com/zhifubaoxiaoyouxi-zromu/ts2ry9/4e26f2203cdb4c74f80c7fd37382ad0f.